src/components/GitPanel.tsx

```diff
@@ -651,10 +656,10 @@ export class GitPanel extends React.Component<IGitPanelProps, IGitPanelState> {
    * List of sorted modified files.
    */
   private get _sortedFiles(): Git.IStatusFile[] {
-    const { files } = this.state;
-
-    files.sort((a, b) => a.to.localeCompare(b.to));
-    return files;
+    const { files, remoteChangedFiles } = this.state;
+    const sfiles = files.concat(remoteChangedFiles);
```

Comment:
It will show up in both the changed and the remote changes sections. I think it should stay this way to convey the different local and the local/remote relationship contexts.